TombEngine/Renderer/RendererDraw.cpp

```diff
@@ -151,43 +148,66 @@ namespace TEN::Renderer
 			_shadowLight->Position :
 			Vector3::Lerp(_shadowLight->PrevPosition, _shadowLight->Position, GetInterpolationFactor());
 
+		static constexpr Vector3 forwardVectors[6] =
```

Comment:
Can we use `Vector3::UnitX` etc here? Available DirectXMath constants are:

```            static const Vector3 Zero;
            static const Vector3 One;
            static const Vector3 UnitX;
            static const Vector3 UnitY;
            static const Vector3 UnitZ;
            static const Vector3 Up;
            static const Vector3 Down;
            static const Vector3 Right;
            static const Vector3 Left;
            static const Vector3 Forward;
            static const Vector3 Backward;

Reply:
Done